ols-geocoder-process/pom.xml

```diff
@@ -54,11 +59,110 @@
 			<version>${project.version}</version>
 			<type>jar</type>
 		</dependency>
+		<dependency>
+			<groupId>ca.bc.gov.ols</groupId>
+			<artifactId>ols-util</artifactId>
+			<version>1.3.0-SNAPSHOT</version>
+		</dependency>
 		<dependency>
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-classic</artifactId>
 			<version>1.2.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.geotools</groupId>
+			<artifactId>gt-referencing</artifactId>
+			<version>${geotools.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.geotools</groupId>
+			<artifactId>gt-epsg-hsql</artifactId>
+			<version>${geotools.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.geotools</groupId>
+			<artifactId>gt-epsg-extension</artifactId>
+			<version>${geotools.version}</version>
+		</dependency>
 	</dependencies>
 
+	<repositories>
+		<repository>
+			<id>osgeo</id>
+			<name>OSGeo Release Repository</name>
+			<url>https://repo.osgeo.org/repository/release/</url>
+			<snapshots><enabled>false</enabled></snapshots>
+			<releases><enabled>true</enabled></releases>
+		</repository>
+	</repositories>
+	
+	<profiles>
+		<profile>
+			<id>StreetPrep</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>exec-maven-plugin</artifactId>
+						<version>1.5.0</version>
+						<executions>
+							<execution>
+								<goals>
+									<goal>exec</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<executable>java</executable>
+							<arguments>
+								<argument>-Xmx3000M</argument>
+								<argument>-classpath</argument>
+									<!-- automatically creates the classpath using all project dependencies, 
+										also adding the project build directory -->
+									<classpath />
+								<argument>ca.bc.gov.ols.streetprep.StreetPrep</argument>
+								<argument>L:\Refractions\MLCSOG\Batch_Geocoder_2011-12\FME2Java\B1 street prep\inputs\</argument>
```

Comment:
I wonder if these 'arguments' are relevant for our purposes. Looks like they are specific to developer's environment

Reply:
Yes, it seems like we probably want those to be configurable somehow, but I'm not sure how. You could just run a java command line in jenkins, instead of a maven target, but the maven exec plugin adds all the necessary classpath entries based on the POM, that's why we are using it. I don't use this to run it myself, so we could just replace these with whatever is required by your build environment, but that doesn't seem very flexible. Open to ideas on how to make this work.